src/components/ExpandableCard.tsx

```diff
@@ -54,96 +52,40 @@ const ExpandableCard = ({
   }
 
   return (
-    <Accordion
-      border="1px"
-      borderColor="border"
-      borderRadius="sm"
-      display="flex"
-      flexDirection="column"
-      marginBottom="4"
-      cursor="pointer"
-      _hover={{
-        backgroundColor: "ednBackground",
-      }}
-      index={isVisible ? [0] : []}
-    >
-      <AccordionItem borderTopWidth="0" flex="1">
-        <Heading as="h3" m={0}>
-          <AccordionButton
-            width="100%"
-            px="6"
-            py="6"
-            flex="1"
+    <>
+      <Accordion type="single" collapsible className="mb-4">
+        <AccordionItem
+          value="item-1"
+          className="rounded-sm border border-gray-200/50 hover:bg-gray-100/50 dark:border-gray-600 dark:hover:bg-gray-800"
+        >
+          <AccordionTrigger
+            hideIcon
             onClick={onClick}
-            _hover={{
-              backgroundColor: "ednBackground",
-            }}
-            _expanded={{
-              backgroundColor: "transparent",
-            }}
+            className="w-full p-6 transition-colors hover:bg-gray-50 hover:text-black md:p-6 dark:hover:bg-gray-800 dark:hover:text-white [&[data-state=open]]:bg-transparent [&[data-state=open]]:text-black dark:[&[data-state=open]]:text-white"
```

Comment:
Similar case with colors, lets simplify this by using some of the DS colors.
```suggestion
            className="w-full p-6 transition-colors hover:bg-background-highlight hover:text-black md:p-6 dark:hover:text-white [&[data-state=open]]:bg-transparent [&[data-state=open]]:text-black dark:[&[data-state=open]]:text-white"
```